drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -708,6 +745,13 @@ int nrf_wifi_mode(const struct device *dev,
 	}
 
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
+
+	k_mutex_lock(&vif_ctx_zep->vif_lock, K_FOREVER);
+	if (!rpu_ctx_zep->rpu_ctx) {
+		LOG_DBG("%s: RPU context not initialized", __func__);
```

Comment:
Should these new log messages be LOG_ERR or LOG_DBG ? I lean towards LOG_DBG as it is not an error case but the format used elsewhere in this document uses LOG_ERR for the same type of cases